packages/strapi-database/lib/constants/index.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 // contentTypes and components reserved names
-const RESERVED_MODEL_NAMES = ['admin', 'boolean', 'date', 'date-time', 'time', 'upload'];
+const RESERVED_MODEL_NAMES = ['admin', 'boolean', 'date', 'date-time', 'format', 'time', 'upload'];
```

Comment:
It'll be a field name, not the model name. I've tested with model names and it's fine.

Reply:
RESERVED_ATTRIBUTE_NAME ??